### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_AI4D:Control type is wrong for "SQL server Big data Cluster"radio button.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7600]
GitHubTags:#SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1,#WCAG4.1.2;#WCAG4.2.1;#DesktopApp;#NVDA;#AI4D#Win32;
Environment Details:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Additional Details:
AI4D
NVDA 2019.2

**Steps to Reproduce :**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster” .
5.Open AI4D and hover on the I accept check box.
**Actual:**
Issue 1.When we hover on the "SQL server Big data Cluster" control type is showing custom which is not correct.
Issue 2. Localize control type for the big data cluster dose  not exist.
****Expected:**
Control type for the "SQL server Big data Cluster" should be radio button.
**User Impact:**
Screen reader User will not able to relate with the control and will misguide with name of the control.
**Attachment for Reference:**
[A11y_AsureDataStudio_Selectthedeploymentoption_AI4D1.pptx](https://github.com/microsoft/azuredatastudio/files/3707399/A11y_AsureDataStudio_Selectthedeploymentoption_AI4D1.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
will be fixed by https://github.com/microsoft/azuredatastudio/pull/8451

**Comment by V-sheman** [comment]
Verified the issue in Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed hence closing the issue.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
